tests/rptest/test_suite_ec2.yml

```diff
@@ -0,0 +1,66 @@
+# TODO work in progress getting clustered mode issues worked out
+ec2:
+  # ignoring bad log line Error while reconciling topics - seastar::abort_requested_exception (abort requested)
+  # e2e test has num_nodes=1 for the redpanda service
+  included:
+    #- tests/tx_verifier_test.py # no tx verifier test
```

Comment:
seems like we should include all tests by default so new tests don't need to consider updating this file. and then we exclude anything that doesn't work on cluster or is being fixed (e.g. bad log lines) then reduce that set over time.